server/src/modules/stats/board/board.stats.controller.ts

```diff
@@ -39,4 +39,20 @@ export class BoardStatsController {
   ): Promise<TimeSpentData[]> {
     return this.boardStatsService.getTimesSpentOnBoardInMs(boardId);
   }
+
+  @Get(':boardId/board-actions-over-time')
+  @UseGuards(JwtAuthGuard)
```

Comment:
Can any authenticated user with a proper board id hit this endpoints? Is it somehow secured besides JwtAuthGuard?

Reply:
No, there is a task for it.